cfg/api.go

```diff
@@ -105,10 +106,13 @@ type API struct {
 
 // CacheConfig configs the cache
 type CacheConfig struct {
-	Type              string   `yaml:"type"`
-	Size              int      `yaml:"size_mb"`
-	MemcachedServers  []string `yaml:"memcachedServers"`
-	DefaultTimeoutSec int32    `yaml:"defaultTimeoutSec"`
+	Type             string   `yaml:"type"`
+	Size             int      `yaml:"size_mb"`
+	MemcachedServers []string `yaml:"memcachedServers"`
+	// TODO (grzkv): This looks to be used as expiration time for cache
+	DefaultTimeoutSec int32  `yaml:"defaultTimeoutSec"`
+	Replicated        bool   `yaml:"replicated"`
```

Comment:
Would it make sense to define this as a different type of cache("replicated")?
That way, we don't have to think about meaning of other combinations(for example: Type=mem, replicated=true)?

Reply:
This is more involved than this. The thing is that the memcache type of cache that already was there has the same problem. You can have type mem + replicas. So, we need to refactor config for it as well. This is not a very small refactoring. I'd do it separately.